#  Always lower-case names
node_frequency: 2.0
names: ["upper","lower"]
address: [16,32]
topics:
    subscribers:
        data: /parallel_port/data
    publishers:
        thruster_stop: /Thruster_Stop
        diagnostics: /diagnostics